concordia/urls.py

```diff
@@ -100,7 +100,11 @@
     path("contact/", views.ContactUsView.as_view(), name="contact"),
     path("act/", views.action_app, name="action-app"),
     path("campaigns/", include(tx_urlpatterns, namespace="transcriptions")),
-    path("reserve-asset/<int:asset_pk>/", views.reserve_asset, name="reserve-asset"),
+    path(
+        "reserve-asset/<int:asset_pk>/<slug:reserve_id>",
```

Comment:
```suggestion
        "reserve-asset/<int:asset_pk>/<slug:reserve_id>/",
```

Reply:
I'm wondering whether this should be in the URL since it doesn't have any meaning and, unlike the asset PKs, is unique to a user. Looking at it, it's not immediately obvious why we're even passing it around since it's stored in the user session and it seems like it could stay on the server side with no exposure to HTML/JS.